src/Tools/ExternalAccess/FSharp/FindUsages/FSharpSourceReferenceItem.cs

```diff
@@ -10,7 +10,9 @@ internal class FSharpSourceReferenceItem
     {
         private readonly Microsoft.CodeAnalysis.FindUsages.SourceReferenceItem _roslynSourceReferenceItem;
 
+#pragma warning disable IDE0051 // Remove unused private members
```

Comment:
nit: How/where is this constructor used?

Reply:
It's not used. But I don't want if F# can be calling it via reflection or something?

Reply:
Might be. Let's keep to be safe